packages/field/src/FormatMixin.js

```diff
@@ -171,9 +171,26 @@ export const FormatMixin = dedupeMixin(
       }
 
       __callParser(value = this.formattedValue) {
-        let result;
-        if (typeof value === 'string') {
-          result = this.parser(value, this.formatOptions);
+        if (value === '') {
+          // Ideally, modelValue should be undefined for empty strings.
+          // For backwards compatibility we return an empty string.
```

Comment:
Can you explain the backwards compatibility? We did not have a value = empty string check before right?
I also would expect the parser to parse `''` --> `''` , is this not the case?

Reply:
No the parser (let's say we have a date) either returns a valid Date or undefined.
Before we introduced Unparseable, it used to return '' indeed. This bugfix restores that behaviors
From a 'conceptual' perspective (whether we have a value in our model) undefined would fit better. Think we can do this small change in the next breaking release)